docs/sphinx/conf.py

```diff
@@ -63,7 +62,7 @@
 # The short X.Y version.
 version = u'0.2'
 # The full version, including alpha/beta/rc tags.
-release = u'0.2.3'
+release = u'0.2.4'
```

Comment:
We can comment these release number lines out if we don't want to have to keep them current (it's easy to overlook them).  How useful is it to have the version number appear in the sphinx documentation? If you pull an old version of the code, you would have to dig through the *rst source files or regenerate the documentation to have the correct version of the docs.